### ISSUE TITLE
Wind

### ISSUE THREAD

**Issue by Jandro22** [studio-minus/ppg-bugs-and-requests#184]
**Feature description**
This would be a setting in Environment, along with rain, snow, and fog, etc. This would apply a customizable amount of "wind" (constant force in one direction) to the entire map. 

**Feature purpose**
The purpose of this is so you can take buildings and other tall objects, and use wind to have them sway more. That isn't the only reason, of course. You could also use it to have humans fly around, or anything else you could think of. This would be one of those general features that can be used for many purposes

**Additional context**
Low wind
![image](https://user-images.githubusercontent.com/53024346/132133229-58f849b5-b7aa-4c91-b8c4-a8cd3eef8f0f.png)
High wind
![image](https://user-images.githubusercontent.com/53024346/132133245-af1a47b7-a534-4603-9886-ca694e2aafd7.png)
A building swaying from the high wind
![image](https://user-images.githubusercontent.com/53024346/132133264-b2a1f647-82ac-41c9-9bd1-eee6a7e2288f.png)

**Comment by SkyHighPurchase** [comment]
Love this idea

**Comment by UniDuki** [comment]
It would also be nice if the wind was stronger higher up, kind of like real life.

**Comment by TamasB14** [comment]
True, we should get the wind option in Enviroment tab. I would also like to see a way for raindrops to extinquish fire.

**Comment by mestiez** [comment]
Wind is a 2D vector though, right? Or can it only go left and right? What about turbulence?

**Comment by Jandro22** [comment]
I would assume that wind would be a 2d vector that a positive value goes to the left and a negative value that goes to the right. If you want it to be more complex you could add an angle for the wind as well (Set angle but for the wind itself). Turbulence would be cool, and I would imagine it could work by having a set range that the wind can be changed by, either less or more.

**Comment by LaCumbiaDelCoronavirus** [comment]
I  am going to add on to this idea, by saying that you should also have the option to have placeable fans as well as some maps having a present "force" for their wind such as Snow because of the fans in the corners.

**Comment by TeslaCoil9001** [comment]
bumparoni

**Comment by ATPGaming** [comment]
are we ever gonna get this in game?

**Comment by DindinYT37** [comment]
> are we ever gonna get this in game?

Please avoid commenting unnecessarily.
